### PR TITLE
Add analysis runner view

### PR DESCRIPTION
**Pull request by danielreti**

### CODE REVIEW

File: web/src/pages/project/ProjectGrid.tsx [hunk 1]
```diff
@@ -97,6 +97,18 @@ const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
         }))
     }
 
+    // return (
```
Comment (illusional) [inline web/src/pages/project/ProjectGrid.tsx:100]: Can this be removed?

File: web/src/pages/project/ProjectSelector.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import { useParams } from 'react-router-dom'
 
 import { Dropdown } from 'semantic-ui-react'
 
 import { useQuery } from '@apollo/client'
 
 import { gql } from '../../__generated__/gql'
 
-import { SearchItem, MetaSearchEntityPrefix } from '../../sm-api/api'
+// import { SearchItem, MetaSearchEntityPrefix } from '../../sm-api/api'
```
Comment (illusional) [inline web/src/pages/project/ProjectSelector.tsx:11]: Remove this too?

**Review by illusional — APPROVED** [review]
Looks good to me, a few nitpicks but not blocking :)

File: test/data/generate_data.py [hunk 3]
```diff
@@ -153,6 +153,31 @@ def generate_random_number_within_distribution():
         for s in sample_ids
     ]
 
+    analyses_to_insert.extend(
+        [
+            AnalysisModel(
+                sample_ids=[s],
```
[the remark this answers is not in the document]
Comment (danielreti) [reply test/data/generate_data.py:159]: Ah I just copied the format of the mock analysis objects you wrote ;)

File: web/src/Links.tsx [hunk 4]
```diff
@@ -1,22 +0,0 @@
-import * as React from 'react'
```
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/Links.tsx:1]: Was a relic from the refactor - the links in this file got separated out into 3 individual link components

File: web/src/Routes.tsx [hunk 5]
```diff
@@ -18,6 +19,25 @@ const Routes: React.FunctionComponent = () => (
...
+            <Route
+                path=":projectName"
+                element={
+                    <ErrorBoundary>
+                        <AnalysisRunnerSummary />
+                    </ErrorBoundary>
+                }
+            />
+        </Route>
+
```
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/Routes.tsx:40]: Ah I'm pretty sure they go rid of this in a previous version of `react-router` to general backlash , I guess they've added it back in - will update

File: web/src/pages/project/ProjectGrid.tsx (hunk 1 above)
Comment (danielreti) [reply web/src/pages/project/ProjectGrid.tsx:100]: Yes

File: web/src/pages/project/ProjectSelector.tsx (hunk 2 above)
Comment (danielreti) [reply web/src/pages/project/ProjectSelector.tsx:11]: Yes

**Pull request merged**